app/models/assignment_repo.rb

```diff
@@ -30,13 +38,13 @@ def private?
   # Public
   #
   def github_team_id
-    repo_access.github_team_id
+    repo_access.present? ? repo_access.github_team_id : nil
```

Comment:
Also, this looks like code for backwards compatibility. If that's the case and we'll :fire: it soon after we ship this, then I think it's ok.